airbyte-integrations/connectors/source-googleanalytics-singer/standardtest/standard_source_test.py

```diff
@@ -22,26 +22,15 @@
 SOFTWARE.
 """
 
-import json
 import pkgutil
 from typing import List
 
 import requests
-from airbyte_protocol import ConfiguredAirbyteCatalog, ConnectorSpecification
-from base_python_test import StandardSourceTestIface
+from base_python_test import DefaultStandardSourceTest
 
 
-class GoogleAnalyticsStandardSourceTest(StandardSourceTestIface):
-    def get_spec(self) -> ConnectorSpecification:
-        raw_spec = pkgutil.get_data(self.__class__.__module__.split(".")[0], "spec.json")
-        return ConnectorSpecification.parse_obj(json.loads(raw_spec))
-
-    def get_config(self) -> object:
-        return json.loads(pkgutil.get_data(self.__class__.__module__.split(".")[0], "config.json"))
-
-    def get_catalog(self) -> ConfiguredAirbyteCatalog:
-        raw_spec = pkgutil.get_data(self.__class__.__module__.split(".")[0], "test_catalog.json")
-        return ConfiguredAirbyteCatalog.parse_obj(json.loads(raw_spec))
+class GoogleAnalyticsStandardSourceTest(DefaultStandardSourceTest):
```

Comment:
@yevhenii-ldv FYI we can completely remove the `standard_tests` directory and `Dockerfile.test` and use a simpler approach for running standard tests by adding the following to the build.gradle: 

```
plugins {
  id 'airbyte-standard-source-test-file'
}

airbyteStandardSourceTestFile {
    // All these input paths must live inside this connector's directory (or subdirectories)
    configPath = "<path to config file to be used in standard tests>" // TODO
    configuredCatalogPath = "<path to configured catalog to be used in tests>" // TODO
    specPath = "<path to spec for use in tests>" // TODO
}
```

This invokes a pre-made Docker image to run tests instead of building `Dockerfile.test` and running it every time we build this connector, which reduces build time. It is also less complicated/more easily understandable than creating a Python test container.

Reply:
See `source-google-adwords-singer` for an example

Reply:
Oh, this is very interesting, didn't know about it, thanks. Applied this technique for this source.

Reply:
I hesitated to mention it because it will change again soon lol. Move fast!